index.bs

```diff
@@ -3174,68 +3242,39 @@ the first existing key in that range.
-    then [=exception/throw=] a "{{TransactionInactiveError}}" {{DOMException}}.
-
-1. Let |range| be the result of
-    [=/converting a value to a key range=] with |query|.
+1. Return the result of [=/creating a request to retrieve multiple items=] with [=ECMAScript/the current Realm record=], [=/this=], "value", |queryOrOptions|, and |count| if given.
     Rethrow any exceptions.
 
-1. Let |operation| be an algorithm to run [=retrieve multiple values from an object store=] with [=ECMAScript/the current Realm record=], |store|, |range|, and |count| if given.
-
-
-1. Return the result (an {{IDBRequest}}) of running [=asynchronously execute a request=] with [=/this=] and |operation|.
-
-NOTE:
-The |query| parameter can be a [=/key=] or [=/key range=] (an {{IDBKeyRange}})
-identifying the [=object-store/record=] values to be retrieved. If null or not given,
-an [=unbounded key range=] is used. If |count| is specified and
-there are more than |count| records in range, only the first |count|
-will be retrieved.
-
 </div>
 
 
 <div algorithm>
 
-The <dfn method for=IDBObjectStore>getAllKeys(|query|, |count|)</dfn> method steps are:
-
-1. Let |transaction| be [=/this=]'s [=object-store-handle/transaction=].
-
-1. Let |store| be [=/this=]'s [=object-store-handle/object store=].
+The <dfn method for=IDBObjectStore>getAllKeys(|queryOrOptions|, |count|)</dfn> method steps are:
 
-1. If |store| has been deleted, [=exception/throw=] an
-    "{{InvalidStateError}}" {{DOMException}}.
+1. Return the result of [=/creating a request to retrieve multiple items=] with [=ECMAScript/the current Realm record=], [=/this=], "key", |queryOrOptions|, and |count| if given.
+    Rethrow any exceptions.
 
-1. If |transaction|'s [=transaction/state=] is not [=transaction/active=],
-    then [=exception/throw=] a "{{TransactionInactiveError}}" {{DOMException}}.
+</div>
 
-1. Let |range| be the result of
-    [=/converting a value to a key range=] with |query|.
-    Rethrow any exceptions.
 
-1. Let |operation| be an algorithm to run [=retrieve multiple keys from an object store=] with |store|, |range|, and |count| if given.
+<div algorithm>
 
-1. Return the result (an {{IDBRequest}}) of running [=asynchronously execute a request=] with [=/this=] and |operation|.
+The <dfn method for=IDBObjectStore>getAllRecords(|options|)</dfn> method steps are:
 
-NOTE:
-The |query| parameter can be a [=/key=] or [=/key range=] (an {{IDBKeyRange}})
-identifying the [=object-store/record=] keys to be retrieved. If null or not
-given, an [=unbounded key range=] is used. If |count| is specified
-and there are more than |count| keys in range, only the first |count|
-will be retrieved.
+1. Return the result of [=/creating a request to retrieve multiple items=] with [=ECMAScript/the current Realm record=], [=/this=], "record", and |options|.
+    Rethrow any exceptions.
 
 </div>
 
+<aside class=advisement>
+  &#x1F6A7;
+  The {{IDBObjectStore/getAllRecords()}} method is new in this edition.
```

Comment:
The other sites that use this wording mention which browsers/versions implement it. I guess the plan is to come back and add that after it actually passes Chromium's I2S?

Also might be good to add something like this for the changed getAll/getAllKeys param.

And finally, maybe "new in this edition" should link to the revision history section?